src/arrays.js

```diff
@@ -1,62 +1,106 @@
 const getNthElement = (index, array) => {
-  // your code here
+  let myIndex = index % array.length;
+  return array[myIndex];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  let text = array.join();
+  return text;
 };
 
 const csvStringToArray = string => {
-  // your code here
+  let text = string.split(',');
+  return text;
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
+//  return array; - I got stuck here, discussed with Jasmin 24/04 and the test is expecting undefined :)
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  // test states we need to return new array
+  let newArray = [];
+  for (let i = 0; i < array.length; i++) {
+    newArray.push(array[i]);
+  }
+  newArray.push(element);
+  return newArray;
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  array.splice(index, 1); // removes 1 element from array, splice is modifying the original array so no need to use return. The "1" is the number of elements to remove
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+  // this converts every number in an array to a string using arrow function (=>)
+  // .map creates a new array
+  // toString() method returns a string representing the object. 
+  return numbers.map(number => number.toString());
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+  // does the same as the above, only this time adding uppercase. The toUpperCase() method returns the value of the string converted to uppercase
+  return strings.map(letter => letter.toString().toUpperCase());
```

Comment:
You don't need `toString()` as it's already a list of strings. I would also rename `letter` to `string`, as it's not an individual letter in your callback function, but the the full string